cst.yml

```
@@ -9,6 +9,8 @@ metadataTest:
   labels:
     - key: 'project'
       value: 'https://github.com/jenkins-infra/docker-packaging'
+    - key: io.jenkins-infra.tools
+      value: "bash,debhelper,fakeroot,git,gpg,gh,jx-release-version,java,jv,jenkins-agent,make"
```

I would put that in a separate PR, nothing to do with updatecli manifest :)

```suggestion
      value: "bash,debhelper,fakeroot,git,gpg,gh,java,jenkins-agent,jv,jx-release-version,make"
```
(sorted)